CONTRIBUTING.md

```diff
@@ -0,0 +1 @@
+{{docs/development/hacking_itk_wasm.md#contributing}}
```

Comment:
Not sure at all that this will work.

Reply:
Since GitHub does not render it, what you think about something like,

```
Please see [our contributing documentation](https://wasm.itk.org/en/latest/development/hacking_itk_wasm.html) for more information.
```

?

Reply:
I made a few attempts in a test repository to see if the transclusion worked but did not succeed. I do not like the cross-reference solution very much, but I adopted it: instead of using the HTML link, I used a relative link directly to the section of interest in the Markdown file at issue.